packages/react-components/react-rating-preview/src/components/RatingDisplay/RatingDisplay.test.tsx

```diff
@@ -1,18 +1,42 @@
 import * as React from 'react';
 import { render } from '@testing-library/react';
-//import { isConformant } from '../../testing/isConformant';
+import { isConformant } from '../../testing/isConformant';
 import { RatingDisplay } from './RatingDisplay';
 
 describe('RatingDisplay', () => {
-  // isConformant({
-  //   Component: RatingDisplay,
-  //   displayName: 'RatingDisplay',
-  // });
-
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
-  it('renders a default state', () => {
-    const result = render(<RatingDisplay>Default RatingDisplay</RatingDisplay>);
-    expect(result.container).toMatchSnapshot();
+  isConformant({
+    Component: RatingDisplay,
+    displayName: 'RatingDisplay',
+    requiredProps: { count: 1160, value: 4.5 },
+  });
+  it('renders the correct number of items', () => {
+    const { getByRole } = render(<RatingDisplay max={10} />);
+    const items = getByRole('img');
+    expect(items.getElementsByClassName('fui-RatingItem').length).toEqual(10);
+  });
+  it('renders the valueText slot when a value is provided', () => {
+    const { getByRole } = render(<RatingDisplay value={3} />);
+    const items = getByRole('img');
+    expect(items.getElementsByClassName('fui-RatingDisplay__valueText').length).toEqual(1);
+  });
+  it('does not render the valueText slot when a value is not provided', () => {
+    const { getByRole } = render(<RatingDisplay />);
+    const items = getByRole('img');
+    expect(items.getElementsByClassName('fui-RatingDisplay__valueText').length).toEqual(0);
+  });
+  it('renders the countText slot when a count is provided', () => {
+    const { getByRole } = render(<RatingDisplay count={1160} />);
```

Comment:
In this case I think you can use the `getByText()` API.

Ex:

```tsx
const { getByText } = render(<RatingDisplay count={1160} />);
const countText = getByText("1160");

expect(countText).toExist() // or maybe .byTruthy()? I can never remember
expect(countText.classList.contains("fui-RatingDisplay__countText").toBeTrue();

```

Reply:
Ah, Sean had the same idea as me :). But I don't think checking the classList is necessary; that's done by conformance tests.
~~`expect(countText.classList.contains("fui-RatingDisplay__countText").toBeTrue();`~~

Reply:
I believe its toBeDefined()

Reply:
> I believe its toBeDefined()

I think that checks for `undefined` but not `null`; but `toBeTruthy()` covers both.